docs/cli.rst

```diff
@@ -1,11 +1,271 @@
+
+
+Reporting Error Logs 
+^^^^^^^^^^^^^^^^^^^^
+
+The ``cci error gist`` command sends the most recent log file to a `GitHub gist <https://docs.github.com/en/github/writing-on-github/creating-gists>`_ so you can quickly and easily share logs with others. For this feature to work you need to ensure that your `github service is set up with the proper scopes <https://cumulusci.readthedocs.io/en/latest/tutorial.html#github-service>`_.
+
+Information included in the gist:
+
+* The current version of ``cci``
+* The current Python version
+* The path to the Python executable
+* The ``sysname`` of the host (such as Darwin)
+* The machine name of the host (such as x86_64)
+* The most recent log file (cci.log) that CumulusCI has created.
+
+The URL for the gist is displayed on the user terminal as output, and a web browser automatically opens a tab to the gist.
+
+
+The ``--debug`` Option
+^^^^^^^^^^^^^^^^^^^^^^
+
+All CumulusCI commands can be passed the ``--debug`` option. When the option is used:
+
+* Any calls to CumulusCI's logger at the debug level are shown.
+* Outgoing HTTP requests are logged.
+* If an error is present, the corresponding stacktrace is shown, and the user is dropped into a `post-mortem debugging <https://docs.python.org/3/library/pdb.html#pdb.post_mortem>`_ session.
+
+.. note:: To exit a debugging session type the command ``quit`` or ``exit``.
+
+
+Log Files
+^^^^^^^^^
+
+CumulusCI creates a log file every time a cci command runs. There are six rotating log files (``cci.log, cci.log1...5``) with ``cci.log`` being the most recent. Log files are stored under ``~/.cumulusci/logs``.
+
+By default, log files document:
+
+* The last command that was entered by the user.
+* All output from the command (including debug information).
+* If a Python-level exception occurs, the corresponding stacktrace is included.
+
+.. note:: If you want debug information regarding the ``requests`` module to be documented in a log file, you must explicitly run the command with the ``--debug`` option.
+
+
+Viewing Stacktraces
+^^^^^^^^^^^^^^^^^^^
+
+If you encounter an error and want more information on what caused it, the ``cci error info`` command displays the last ``n`` lines of the stacktrace (if present) from the last command executed in CumulusCI. (Note that this a Python stacktrace showing where CumulusCI encountered an error.)
+
+Additionally, there is a ``--max-lines`` option to limit the number of lines of stacktrace shown.
+
+
+Seeing Stacktraces Automatically
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If you would like to investigate bugs in CumulusCI when you find them, set the config option ``show_stacktraces`` to ``True`` in the ``cli`` section of ``~/.cumulusci/cumulusci.yml``. Afterward, stacktraces are no longer suppressed when they are thrown within CumulusCI.
+
+Usage Errors (wrong command line arguments, missing files, and so on) don't show exception tracebacks because they are seldom helpful in that case.
```

Comment:
```suggestion
Usage errors (such as incorrect command line arguments, missing files, and so on) don't show exception tracebacks because they are seldom helpful in that case.
```